packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -394,14 +394,15 @@ export class CdkToolkit {
 
   private async selectStacksForDiff(stackNames: string[], exclusively?: boolean) {
```

Comment:
`selectStacksForDiff` ? Is that just a poorly chosen name?

It's a bit confusing in this context.

Reply:
The same stacks are selected for diff and synthesis. This method is reused in both contexts.